Makefile

```diff
@@ -1,6 +1,6 @@
 DISABLE_CACHE = 'false'
 
-ARCH = rpi# rpi/amd64/i386/armv7hf/armel
+ARCH = rpi# rpi/amd64/i386/armv7hf/armel/armhf-alpine/amd64-alpine/i386-alpine
```

Comment:
We'd also have to use separate builds for rpi and armv7hf even though it's the same thing, but I don't see that as too much of a problem.